Formula/n/netatalk.rb

```diff
@@ -0,0 +1,72 @@
+class Netatalk < Formula
+  desc "File server for Macs, compliant with Apple Filing Protocol (AFP)"
+  homepage "https://netatalk.io"
+  url "https://github.com/Netatalk/netatalk/releases/download/netatalk-4-0-8/netatalk-4.0.8.tar.xz"
+  sha256 "d09d591521b27b91b1c2a6255a2e059af8c9fda43570f983f0a145706e5f7628"
+  license all_of: [
+    "GPL-2.0-only",
+    "GPL-2.0-or-later",
+    "LGPL-2.0-only",
+    "LGPL-2.1-or-later",
+    "BSD-2-Clause",
+    "BSD-3-Clause",
+    "MIT",
+  ]
+  head "https://github.com/Netatalk/netatalk.git", branch: "main"
+
+  depends_on "docbook-xsl" => :build
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkgconf" => :build
+
+  depends_on "berkeley-db@5" # macOS bdb library lacks DBC type etc.
+  depends_on "cracklib"
+  depends_on "libevent"
+  depends_on "libgcrypt"
+  depends_on "mysql"
+  depends_on "openldap" # macOS LDAP.Framework is not fork safe
+
+  uses_from_macos "libxslt" => :build
+  uses_from_macos "perl"
+
+  on_linux do
+    depends_on "avahi" # on macOS we use native mDNS instead
+    depends_on "cups" # used by the AppleTalk print server
+    depends_on "dbus"
+  end
+
+  # The below are ports of upstream netatalk patches:
+  # To be removed when integrating the stable release after 4.0.8
+
+  patch do
+    url "https://github.com/Netatalk/netatalk/commit/dc9bcd780ada457db0f8cc48c73dd2fd083b0f3f.patch?full_index=1"
+    sha256 "ad8b4332aaaad62424acabf52d0d19a497a47d2fae73aa794c1bdbeb0e482a9c"
+  end
+
+  patch do
+    url "https://github.com/Netatalk/netatalk/commit/5aca546bac3cda9ae7eaadacf0823ffa659f9f31.patch?full_index=1"
+    sha256 "7ac878414d0d476cb8f42f5bc0cc767d3495407b6b23f1b601f206a62f757600"
+  end
+
+  def install
+    args = [
```

Comment:
Try doing
```ruby
ENV.append "LDFLAGS", "-L#{Formula["berkeley-db@5"].opt_lib}"
```
at the start of the `install` method.

This might need to be paired with a `with-bdb-path` flag too.

Reply:
This is what you were thinking? https://github.com/Homebrew/homebrew-core/pull/202713/commits/a9569be53f838d6128b1861cdf896dce22aedbe5

Reply:
Yes, but that doesn't help, unfortunately. Let's try something else...